Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -637,7 +637,7 @@ - (BOOL)isFlipped
 @end
 
 @interface WKResponderChainSink : NSResponder {
-    NSResponder *_lastResponderInChain;
+    RetainPtr<NSResponder> _lastResponderInChain;
```

Comment:
I am a little worried about reference cycles here, maybe we can use a WeakObjCPtr here?